admin/app/view_models/workarea/admin/bulk_action_product_edit_view_model.rb

```diff
@@ -18,13 +18,43 @@ def selected_false?(hash, field)
       end
 
       def pricing_prices
-        pricing.fetch('prices', []).first || {}
+        pricing['prices'] || {}
       end
 
       def segments
         return [] if settings['active_segment_ids'].blank?
         @segments ||= Segment.in(id: settings['active_segment_ids']).to_a
       end
+
+      def pricing_actions
+        [
+          [
+            t('workarea.admin.bulk_action_product_edits.pricing.set'),
+            'set'
+          ],
+          [
+            t('workarea.admin.bulk_action_product_edits.pricing.increase'),
+            'increase'
+          ],
+          [
+            t('workarea.admin.bulk_action_product_edits.pricing.decrease'),
+            'decrease'
+          ]
+        ]
+      end
+
+      def pricing_types(currency)
```

Comment:
Why pass in currency here, isn't it always going to be `Money.default_currency`?